doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_custom_pcb.rst

```diff
@@ -75,17 +76,328 @@ This is useful when making a change on the PCB (for example, when changing the c
+
+      .. code-block:: json
+
+         {
+           "power": {
+             "scheme": "VDD_VDDH_1V8"
+           }
+         }
+
+Inductor configuration
+----------------------
+
+Each supported supply scheme includes an inductor.
+No additional configuration is needed beyond setting the ``power->scheme`` property.
+
+GPIO power configuration
+------------------------
+
+GPIO port assignments are specified within the ``ioPortPower`` object, with each mode explicitly indicated.
+
+The available port configuration keys are the following:
+
++------+--------------------+
+| Port | Port configuration |
+|      | key                |
++======+====================+
+| P1   | p1Supply           |
++------+--------------------+
+| P2   | p2Supply           |
++------+--------------------+
+| P6   | p6Supply           |
++------+--------------------+
+| P7   | p7Supply           |
++------+--------------------+
+| P9   | p9Supply           |
++------+--------------------+
+
+The supported operating modes for these ports are the following:
+
++--------------------------+--------------------+
+| Supported operating mode | JSON configuration |
++==========================+====================+
+| Disconnected             | DISCONNECTED       |
++--------------------------+--------------------+
+| Shorted                  | SHORTED            |
++--------------------------+--------------------+
+| External1v8              | EXTERNAL_1V8       |
++--------------------------+--------------------+
+| ExternalFull             | EXTERNAL_FULL      |
++--------------------------+--------------------+
+
+The resulting JSON configuration is structured as follows:
+
+.. code-block:: json
+
+   {
+     "ioPortPower": {
+       "p1Supply": "EXTERNAL_1V8",
+       "p2Supply": "EXTERNAL_1V8",
+       "p6Supply": "EXTERNAL_1V8",
+       "p7Supply": "EXTERNAL_1V8",
+       "p9Supply": "EXTERNAL_FULL"
+     }
+   }
+
+GPIO power drive and impedance configuration
+--------------------------------------------
+
+This section specifies the IO port impedance settings for **P6** and **P7**.
+
+The nRF54H20 SoC allows you to select from these supported impedance values:
+
+* 33 Ohms
+* 40 Ohms
+* 50 Ohms
+* 66 Ohms
+* 100 Ohms
+
+The configuration can be represented as follows:
+
+.. code-block:: json
+
+   {
+     "ioPortImpedance": {
+       "p6ImpedanceOhms": 50,
+       "p7ImpedanceOhms": 50
+     }
+   }
+
+Low Frequency Oscillator (LFOSC) configuration
+----------------------------------------------
+
+The JSON format is inside ``lfosc: { lfxo: { ... } }``.
+
+The following parameters are used in the LFOSC configuration:
+
+* ``accuracyPPM`` - Specifies the accuracy of the oscillator in parts per million (PPM).
+  Supported values range from 20 to 500 PPM.
+* ``mode`` - Defines the oscillator mode. Can be ``CRYSTAL`` for crystal operation, ``EXT_SINE`` for external sine wave input, or ``EXT_SQUARE`` for external square wave input.
```

Comment:
```suggestion
* ``mode`` - Defines the oscillator mode.
  It can be ``CRYSTAL`` for crystal operation, ``EXT_SINE`` for external sine wave input, or ``EXT_SQUARE`` for external square wave input.
```